scripts/quests/windurst/A_Pose_by_Any_Other_Name.lua

```diff
@@ -137,6 +143,12 @@ quest.sections =
             ['Angelica'] =
             {
                 onTrigger = function(player, npc)
+                    if xi.events.starlightCelebration.isStarlightEnabled() ~= 0 then
+                        if xi.events.starlightCelebration.npcGiftsNpcOnTrigger(player, 4) then
+                            return
```

Comment:
Can these have a comment to say that you're short-circuiting?

Reply:
Yes--will add.

Reply:
Added in latest commit!